src/main/java/org/jenkinsci/test/acceptance/po/ConfigurablePageObject.java

```diff
@@ -45,6 +45,10 @@
  * @author ogondza.
  */
 public abstract class ConfigurablePageObject extends PageObject {
+
+    private static final By SAVE_BUTTON = By.xpath(
+            "//div[contains(@class, 'bottom-sticker-inner')]//input[@type='submit'] | //div[contains(@class, 'bottom-sticker-inner')]//button[contains(@name, 'Submit')]");
```

Comment:
The latter clause does not work for `f:submit` when `name` is overridden. I think it would be more reliable to look for the class `jenkins-submit-button`?